felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -142,6 +148,12 @@ type bpfInterface struct {
 	dpState bpfInterfaceState
 }
 
+func (i *bpfInterfaceState) clearPolicies() {
```

Comment:
I think, whenever this is called, we need to put the indexes back in the freelist but I can't see where that happens right now.

Reply:
was called after updateIfaceStateMap, but moved it inside to where the indexes are returned